### ISSUE TITLE
Breaking changes?

### ISSUE THREAD

**Issue by lekoaf** [digitalbazaar/forge#812]
What happened?

CHANGELOG.md - Release 0.10.0. - 7 hours ago

And when you take a look at the CHANGELOG.md:
0.10.0 - 2019-09-01

Are you a year off in the changelog?

And why are there breaking changes in a minor version?

Because of this, the renovate bot tries to "sneak" this through with all the other minor updates in our app.

**Comment by davidlehn** [comment]
> [...]
> 0.10.0 - 2019-09-01
> 
> Are you a year off in the changelog?
> 

I goofed the date.  Fixed.


> And why are there breaking changes in a minor version?
> 
> Because of this, the renovate bot tries to "sneak" this through with all the other minor updates in our app.

It's using the common practice of treating bumps of 0.x as major breaking releases.  Normal npm usage special cases 0.x.y and should handle this just fine.  I don't know anything about that bot.  Considering the age of this library we really should just call it 1.0.0 soon. ;-)  At that point it would use the normal major bumps.

In any case, I suspect no one has ever used the path functions that were removed, so your code will likely keep working just fine.

**Comment by lekoaf** [comment]
Fair enough. Renovate bot is like a Greenkeeper for your dependencies. I thought it was widely used? https://github.com/renovatebot/renovate
